### DISCUSSION TITLE
500 Internal Server Error

### DISCUSSION THREAD

**Discussion by jboeck** [Part-DB/Part-DB-server#284] [Q&A]
Hi,
bräuchte bitte Hilfe. Ich wollte auf das aktuelle Docker-image updaten. Seither bekomme ich die Fehlermeldung
**500 Internal Server Error**
part-db läuft ...oder lief... bei mir auf einer Synology DS.  Die Einstellungen des Containers sind m.E. ok bzw. so wie sie waren.
Ich bitte um Nachsicht wenn meine Angaben ungenau sind, part-db ist mein erster Kontakt mit Docker und datenbank.
Danke, Jan

**Comment by jbtronics** [comment]
Vermutlich fehlt da die Datenbank Migration. Dann musst du den `docker exec --user=www-data partdb php bin/console doctrine:migrations:migrate` Befehl wie hier beschrieben ausführen: https://docs.part-db.de/installation/installation_docker.html#update-part-db

**Reply by jboeck** [reply]
Vielen Dank 👍
